demo/update2.json

```diff
@@ -1,9 +1,6 @@
 {
     "selector": "ez-platform-ui-app",
     "update": {
-        "attributes": {
```

Comment:
same remark as in https://github.com/ezsystems/hybrid-platform-ui/pull/7/files#r117753993